src/main/java/org/opentripplanner/model/plan/Itinerary.java

```diff
@@ -218,10 +221,27 @@ public Optional<Leg> firstTransitLeg() {
     return legs.stream().filter(Leg::isTransitLeg).findFirst();
   }
 
+  public Float accessibilityScore() {
+    var scores = legs
+      .stream()
+      .map(Leg::accessibilityScore)
+      .filter(Objects::nonNull)
+      .mapToDouble(Float::doubleValue)
+      .toArray();
+
+    var sum = Arrays.stream(scores).sum();
+    if (scores.length > 0) {
+      return (float) sum / scores.length;
```

Comment:
Again, this would change the scale, would it not? And if you wanted to use a floor/ceiling I think using the lowest value makes most sense since you want to show the worst value to the user.